### PR TITLE
Fixed a Bug in the Tray Seeding Python Program

### LINKED ISSUE #338: Seeding Report: Incorrect Displayed Value
The Tray Seeding Summary at the bottom of a seeding report will display the value "NaN" for both the "Total Number of Tray Seeds Planted" and "Average Seeds Planted per Hour" at certain intervals. This should be displaying an actual sum of the correct values. The "NaN" value seems to show up occasionally when searching within the year 2019.

In order to reproduce this bug:
1. Log in to FarmData2
2. Click on the BarnKit tab
3. Click on Seeding Report
4. Enter an interval of time within 2019 of atleast a few months(3-4).

Observed Result: Values report "NaN"

image

Desired Result: Values are filled in

image

### PR DESCRIPTION
**Pull request by FutzMonitor**
__Pull Request Description__

Closes #338. Before the Tray Seeding Table had some `NaN` values. 

image

After closer inspection, using the CSV export functionality, it seems that some of the logs had the value `false` under their _Seedings_ and _Cells/Tray_ column. Frankly, this may not serve as a final solution but mainly as a way of moving forward to isolate this potential bug. The only difference I noticed between the two lines that I changed in the traySeeding.csv and the rest of the lines is that those two had "0 seedings". I changed that and rebuilt the database and now all those columns have numbers like they're supposed to: 

image

~~**NOT ready to be merged into main**~~
~~Further discussions on what is truly going on to hopefully come to an understanding of the underlying issue.~~

It seems that the culprit of this bug where logs where the total seedings and cells/flat were 0. An additional case was added to account for scenarios where cells/flat is 0 and some pre-existed code was modified to allow for a conditional statement to handle scenarios were seeding was 0. 

---
__Licensing Certification__

FarmData2 is a [Free Cultural Work](https://freedomdefined.org/Definition) and all accepted contributions are licensed as described in the LICENSE.md file. This requires that the contributor holds the rights to do so. By submitting this pull request __I certify that I satisfy the terms of the [Developer Certificate of Origin](https://developercertificate.org/)__ for its contents.

### CODE REVIEW

File: docker/sampleDB/addTraySeedings.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def addSeedings(row, plantingID, seedingTypeID):
         # If seed count not given, assume one seed per tray cell.
         cellsPerFlat = int(row[6])
```
Comment (FutzMonitor) [inline docker/sampleDB/addTraySeedings.py:90]: Adding something here could potentially solve the issue.

**Comment by braughtg** [comment]
Eventually we will want this PR to not have made any changes to the .csv file directly.  Instead all of the changes should be made in the .py files that process the .csv files and turn them into FarmData2 logs and assests.

File: docker/sampleDB/addTraySeedings.py [hunk 2]
```diff
@@ -85,11 +85,14 @@ def addSeedings(row, plantingID, seedingTypeID):
...
+    if type(totalSeeds) != int or type(totalSeeds) != float:
+        totalSeeds = 0
+
     if totalSeeds == 0:  
         # If seed count not given, assume one seed per tray cell.
         cellsPerFlat = int(row[6])
         totalSeeds = totalFlats*cellsPerFlat
-   
+
     seedsPerFlat = totalSeeds/totalFlats
```
Comment (FutzMonitor) [inline docker/sampleDB/addTraySeedings.py:97]: Check if there are 0's

**Comment by FutzMonitor** [comment]
This PR is ready to be reviewed.

**Pull request merged**